samples/cpp/benchmark_app/README.md

```diff
@@ -155,6 +155,18 @@ Options:
     -pc                         Optional. Report performance counters.
     -dump_config                Optional. Path to JSON file to dump IE parameters, which were set by application.
     -load_config                Optional. Path to JSON file to load custom IE parameters. Please note, command line parameters have higher priority than parameters from configuration file.
+                                Example 1: a simple JSON file for HW device with primary properties.
+                                          {
+                                             "CPU": {"NUM_STREAMS": "3", "PERF_COUNT": "NO"}
+                                          }
+                                Example 2: a simple JSON file for meta device(AUTO/MULTI) with HW device properties.
+                                          {
+                                             "AUTO": {
+                                                      "PERFORMANCE_HINT": "", "PERF_COUNT": "NO",
+                                                      "CPU": "NUM_STREAMS 3 INFERENCE_NUM_THREADS 5",
+                                                      "GPU": "NUM_STREAMS 5"
+                                             }
+                                          }
```

Comment:
Why not something like:
```
{
   "AUTO": {
            "PERFORMANCE_HINT": "", "PERF_COUNT": "NO",
            "DEVICE_PROPERTIES" : {
                "CPU": { "NUM_STREAMS" : "3", "INFERENCE_NUM_THREADS" : "5" },
                "GPU": { "NUM_STREAMS" : "5"}
            }
     }
}
```
It will allow to keep config json friendly. Otherwise we can potentially have problem with properties which string type, for example

Reply:
> @nkogteva we would like to decouple "DEVICE_PROPERTIES" and secondary properties for target HW devices.
> 
> Then user can use one configuration jason file included all secondary properties of all candidate devices (like CPU, GPU, VPU). And use "DEVICE_PROPERTIES" to select candidate device and debug behavior one by one.

My question is mostly about file format itself. The following line ```"CPU": "NUM_STREAMS 3 INFERENCE_NUM_THREADS 5"``` for AUTO configuration file looks dangeres for me from parsing perspective.

Reply:
@nkogteva I see. Could we use below format?
```
{
   "AUTO": {
            "PERFORMANCE_HINT": "", 
            "PERF_COUNT": "NO",
            "DEVICE_PROPERTIES" : "CPU",
            {
                "CPU": { "NUM_STREAMS" : "3", "INFERENCE_NUM_THREADS" : "5" },
                "GPU": { "NUM_STREAMS" : "5"}
            }
     }
}
```

Reply:
@wangleis json not seems to be valid. You can use online json validator

Reply:
> Why not something like:
> 
> ```
> {
>    "AUTO": {
>             "PERFORMANCE_HINT": "", "PERF_COUNT": "NO",
>             "DEVICE_PROPERTIES" : {
>                 "CPU": { "NUM_STREAMS" : "3", "INFERENCE_NUM_THREADS" : "5" },
>                 "GPU": { "NUM_STREAMS" : "5"}
>             }
>      }
> }
> ```
> 
> It will allow to keep config json friendly. Otherwise we can potentially have problem with properties which string type, for example

Got your point. And I agree.

Reply:
Updated.